indico/modules/users/models/users.py

```diff
@@ -38,13 +38,14 @@
 
 
 class UserTitle(RichIntEnum):
-    __titles__ = ('', _('Mr'), _('Ms'), _('Mrs'), _('Dr'), _('Prof.'))
+    __titles__ = ('', _('Mr'), _('Ms'), _('Mrs'), _('Dr'), _('Prof.'), _('Mx'))
     none = 0
     mr = 1
     ms = 2
     mrs = 3
     dr = 4
     prof = 5
+    mx = 6
```

Comment:
Is it better to have it at the end of the list or would it be more inclusive to add it after Mrs? 
Semantically it would make more sense to add it after Mrs to keep the non-academic titles and academic titles separated.
I am not sure if it makes much difference, though.

Reply:
Not possible without updating all existing values in the DB :/
But the `sorted=True` for the form field will use alphabetical order for the dropdown options, so the order is blank-dr-m*-prof which isn't perfect but better than having the non-academic titles separated ;)

Reply:
actually, I think in python 3 this will be possible since the `__dict__` of a class is ordered there

```python
>>> class MyEnum(int, Enum):
...     a = 2
...     b = 1
...     c = 3
...

>>> list(MyEnum)
[<MyEnum.a: 2>, <MyEnum.b: 1>, <MyEnum.c: 3>]

```